### PR TITLE
Adding ipaas-profile training content

### PR DESCRIPTION
**Pull request by kavishkafernando**
## Purpose
Adding ipaas-profile training content

### CODE REVIEW

File: ipaas-profile/dbtosf_project/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+org = "kavi"
```
Comment (ramith) [inline ipaas-profile/dbtosf_project/Ballerina.toml:2]: shall we change the org to something else (something meaningful)

File: ipaas-profile/dbtosf_project/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+name = "dbtosf_project"
```
Comment (ramith) [inline ipaas-profile/dbtosf_project/Ballerina.toml:3]: @a5anka do we need to upgrade this to latest ballerina version?

File: ipaas-profile/dbtosf_project/Ballerina.toml (hunk 1 above)
Comment (kavishkafernando) [reply ipaas-profile/dbtosf_project/Ballerina.toml]: updated

File: ipaas-profile/dbtosf_project/Ballerina.toml (hunk 2 above)
Comment (kavishkafernando) [reply ipaas-profile/dbtosf_project/Ballerina.toml:3]: These template files are generated by Choreo when we select the Ballerina project preset

**Review by ramith — APPROVED** [review]

**Pull request merged**